Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -466,15 +466,19 @@ static CTFrameRef _DWriteGetFrame(CFAttributedStringRef string, CFRange range, C
         // Iterate through runs on the current line
         for (j = i; (j < numOfGlyphRuns) && (glyphRunDetails._baselineOriginY[j] == baselineOriginYForCurrentLine); ++j) {
             // Create _CTRun objects and make them part of _CTLine
-            _CTRun* run = [[_CTRun new] autorelease];
+            __CTRun* run = const_cast<__CTRun*>(_CTRunCreate());
```

Comment:
if you look below, you are accessing members of the struct, if it's CTRunRef, it would end up being a const ref.
can't assign to a const ref :p

---
In reply to: [111455427](https://github.com/Microsoft/WinObjC/pull/2511#discussion_r111455427) [](ancestors = 111455427)